src/njoy21/input/DTFR/Card4/HollerithNames.hpp

```diff
@@ -0,0 +1,18 @@
+struct HollerithNames {
+  using Value_t = std::vector<std::string >;
+  static std::string name(){ return "hollerithNames"; }
+
+  static std::string description() {
+    return 
+      "HollerithNames is a vector of the six-character hollerith names for\n"
+      "edits. User may enter names for as many cards as needed. Ther will be\n"
+      "(iptotl - 3) names read. Each name is delimited with a '.";
+  }
+
+  static bool verify( const Value_t& hollerithNames, int nConstants ){
```

Comment:
Why would you pass in Ned? The requirement is that you have ( iptotl - 3 ) Hollerith Names, and Ned can be greater than ( iptotl - 3 ). 
And I tried to just pass in iptotl into the verify function, but then a vector of length iptotl is constructed, and the system gets sad that I'm only giving it ( iptotl - 3 ).